PyInstaller/hooks/pre_find_module_path/hook-distutils.py

```diff
@@ -26,8 +26,15 @@
 def pre_find_module_path(api):
     # Absolute path of the system-wide "distutils" package when run from within
     # a venv or None otherwise.
-    distutils_dir = getattr(distutils, 'distutils_path', None)
-    if distutils_dir is not None:
+
+    # opcode is not a virtualenv module, so we can use it to find the stdlib
+    # Technique taken from virtualenv's "distutils" package detection at
+    # https://github.com/pypa/virtualenv/blob/0ab032ce1a/virtualenv_embedded/distutils-init.py#L5
+    import opcode
+
+    system_module_path = os.path.normpath(os.path.dirname(opcode.__file__))
+    loaded_module_path = os.path.normpath(os.path.dirname(distutils.__file__))
+    if system_module_path != loaded_module_path:
         # Find this package in its parent directory.
-        api.search_dirs = [os.path.dirname(distutils_dir)]
-        logger.info('distutils: retargeting to non-venv dir %r' % distutils_dir)
+        api.search_dirs = [system_module_path]
+        logger.info('distutils: retargeting to non-venv dir %r' % system_module_path)
```

Comment:
` % ` her hill format the string, even if it will nevver be written to the log. This is a common mistake.
Replace it by `, `, thus changing the remainder into additional arguments.